.circleci/config.yml

```diff
@@ -6,12 +6,12 @@ orbs:
 executors:
   dockerizer:
     docker:
-      - image: cimg/go:1.17.6
+      - image: cimg/go:1.18.7
```

Comment:
lotus and lily are now on go version 1.18, ci must match; mechanical change in this file.